src/BackendBuilder.ts

```diff
@@ -39,11 +50,11 @@ export class BackendBuilder {
   private async generateResolvers() {
     const modules: { [id: string]: any } = {};
 
-    for (const resolver of this.backend.resolvers.types) {
-      modules[`./generated/${resolver.name}`] = sourceModule(
-        transpile(resolver.output)
+    this.backend.resolvers.types.map((item) => {
```

Comment:
I think is better to keep the `for` loop, an alternative could be the `foreEach` method because the `map()` doesn't make sense if you don't return a new type

Reply:
You're right, thanks